zerver/tests/test_ldap.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+from django.contrib.auth import authenticate
+from django.test.utils import override_settings
+
+from zerver.lib.test_classes import ZulipTestCase
+from zerver.models import get_realm
+from zproject.backends import ZulipLDAPAuthBackend, ZulipLDAPExceptionOutsideDomain
+
+from django_auth_ldap.config import LDAPSearch
+
+import ldap
+import mock
+
+"""
+This is a file for additional LDAP tests, that don't belong
+anywhere else.
+"""
+
+class DjangoToLDAPUsernameTests(ZulipTestCase):
+    def setUp(self) -> None:
+        super().setUp()
```

Comment:
This super().setUp() is actually useless, it's a leftover from when things were based on the LDAPTestCase class, which did have its own setup() that needed to be called by subclasses. I'll get rid of this here.

Reply:
But this whole thing made me notice a more important thing - tearDown(). We actually do have some tearDown() code in ZulipTestCase and I think we should make sure that any tests that override tearDown remember to run super().tearDown() - made a PR with more elaboration at https://github.com/zulip/zulip/pull/13300 @timabbott